tests/elemental/installation.pm

```diff
@@ -0,0 +1,97 @@
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Test boot of Elemental OS
+# Maintainer: elemental@suse.de
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+
+use testapi;
+use power_action_utils qw(power_action);
+use serial_terminal qw(select_serial_terminal);
+
+sub run {
+    my ($self) = @_;
+    my $cloudconfig = "/tmp/cloud-config.yaml";
+    my %boot_entries = (
+        active => {
+            tag => 'elemental-bootmenu-default'
+        },
+        passive => {
+            tag => 'elemental-bootmenu-fallback'
+        },
+        recovery => {
+            tag => 'elemental-bootmenu-recovery'
+        }
+    );
+
+
+    # Wait for GRUB
+    $self->wait_grub(bootloader_time => 120);
+    send_key('ret', wait_screen_change => 1);
+    wait_still_screen(timeout => 90);
+    save_screenshot();
+
+    ## No GUI, easier and quicker to use the serial console
+    select_serial_terminal();
+
+    # Add a simple cloud-config
+    my $rootpwd = script_output('openssl passwd -6 ' . get_var('TEST_PASSWORD'));
+    assert_script_run('curl ' . data_url('elemental/cloud-config.yaml') . ' -o ' . $cloudconfig);
+    assert_script_run("sed -i 's|%TEST_PASSWORD%|$rootpwd|g' $cloudconfig");
```

Comment:
There's `file_content_replace()` in `lib/utils.pm` which does this.

Reply:
Yes I forgot this one... I already used it back in the day on HANA test :D